front/plugins/_publisher_telegram/tg.py

```
@@ -79,45 +80,50 @@ def check_config():
-        mylog('debug', [req])
-
-        p = subprocess.Popen(req, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True)
-        stdout, stderr = p.communicate()
-
-        # write stdout and stderr into .log files for debugging if needed
-        # Log the stdout and stderr
-        mylog('debug', [stdout, stderr])
-
-        # log result
-        result = stdout
+        proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)